src/cpp/util/time.cc

```diff
@@ -59,6 +60,20 @@ void Timepoint2Timespec(const system_clock::time_point& from,
   to->tv_nsec = nsecs.count();
 }
 
+void TimepointHR2Timespec(const high_resolution_clock::time_point& from,
+                          gpr_timespec* to) {
+  high_resolution_clock::duration deadline = from.time_since_epoch();
```

Comment:
What should be const? deadline? Do we care? I essentially copied the existing Timepoint2Timespec code.